src/main/resources/Message.properties

```diff
@@ -1,3 +1,9 @@
+ACCEPT_DEPENDENT_EMAIL_FOOTER = You are receiving this email because you have been selected to be a trusted companion.
+ACCEPT_DEPENDENT_EMAIL_GREETING = would like you to be a trusted companion.
```

Comment:
Insert and use a placeholder for the user name.

Reply:
Updated in: https://github.com/ibi-group/otp-middleware/pull/256/commits/4a00bfa58c2f747e1f645b4b2b0a4739ac136b6f